### PR TITLE
post the blog for Kata Containers practice in Istio

### PR DESCRIPTION
**Pull request by zhlsunshine**
post the blog for Kata Containers practice in Istio.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: .spelling [hunk 1]
```diff
@@ -863,6 +868,7 @@ RPC
 RPCs
 runtime
 runtimes
+RuntimeClass
```
Comment (ericvn) [inline .spelling:871]: RuntimeClass should likely be backpacked and not added here.

File: .spelling [hunk 2]
```diff
@@ -229,6 +230,8 @@ ConfigMap
 configmaps
 configurability
 containerID
+Containerd
+Containers
```
[the remark this answers is not in the document]
Comment (zhlsunshine) [reply .spelling:234]: Yeah, container is okay, but `Container` and `Containers` are not okay

File: .spelling (hunk 1 above)
Comment (zhlsunshine) [reply .spelling:871]: Hi @ericvn okay, done!

**Comment by kfaseela** [comment]
@ericvn : should we enforce https://istio.io/latest/docs/releases/contribute/diagrams/ for newly added diagrams?

File: .spelling (hunk 2 above)
[the remark this answers is not in the document]
Comment (zhlsunshine) [reply .spelling]: Hi @ericvn, I have removed `Containers` dict。

[the remark this answers is not in the document]
Comment (zhlsunshine) [reply .spelling]: Hi @ericvn, I have removed `Containers` dict.

**Comment by ericvn** [comment]
> @ericvn : should we enforce https://istio.io/latest/docs/releases/contribute/diagrams/ for newly added diagrams?

Certainly, I think we want SVGs. However, I think your are probably referencing the container diagram. We have allowed other formats recently (thinking Solo.io), but I do agree it would be good to follow the guidelines.

**Comment by zhlsunshine** [comment]
Hi @ericvn & @kfaseela, one more question: shall I need to update my container diagram?

**Comment by kfaseela** [comment]
> Hi @ericvn & @kfaseela, one more question: shall I need to update my container diagram?

@zhlsunshine you can do it once the major review for the content is done, and the content is ready for approval. Who should review this from other WGs other than documentation? could you please add them to the review?

**Comment by kfaseela** [comment]
Also instead of the png files for outputs, can't we have a bash snippet as we do usually, so that we can remove your username and things from outside and make it look generic?

File: content/en/blog/2023/workload-injection-with-kata/index.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
+---
+title: "Inject workload with Kata Containers in Istio"
+description: "Inject workload with Kata Containers in Istio, instead of runc."
+publishdate: 2023-03-01
+attribution: "Steve(Huailong) Zhang (Intel)"
+keywords: [Istio, Kata, security]
+---
+
+[Kata Containers](https://github.com/kata-containers), which  is a merger of two existing open source projects: `Intel Clear Containers` and `Hyper runV`, is an open source project and community working to build a standard implementation of lightweight Virtual Machines (VMs) that feel and perform like containers, but provide the workload isolation and security advantages of VMs.
```
Comment (rootsongjc) [inline content/en/blog/2023/workload-injection-with-kata/index.md:9]:
```suggestion
[Kata Containers](https://github.com/kata-containers), which is a merger of two existing open source projects: `Intel Clear Containers` and `Hyper runV`, is an open source project and community working to build a standard implementation of lightweight Virtual Machines (VMs) that feel and perform like containers, but provide the workload isolation and security advantages of VMs.
```

**Comment by zhlsunshine** [comment]
> @zhlsunshine you can do it once the major review for the content is done, and the content is ready for approval. Who should review this from other WGs other than documentation? could you please add them to the review?

Hi @kfaseela, there is no particular one need to review this from other WGs, and it's open for all community. Thanks!

**Comment by zhlsunshine** [comment]
> Also instead of the png files for outputs, can't we have a bash snippet as we do usually, so that we can remove your username and things from outside and make it look generic?

Hi @kfaseela, got it, I will remove the bash snippet, and just the outputs in the png.

**Review by howardjohn — CHANGES_REQUESTED** [review]
I am not very comfortable with this blog post. It essentially is an advertisement for Kata containers and is almost completely unrelated to Istio. This is even explicitly called out in the conclusion:

"Basically, there is no any difference for using Kata Containers with other generic container runtimes in Istio."

**Comment by ericvn** [comment]
> I am not very comfortable with this blog post. It essentially is an advertisement for Kata containers and is almost completely unrelated to Istio. This is even explicitly called out in the conclusion:
> 
> "Basically, there is no any difference for using Kata Containers with other generic container runtimes in Istio."

@istio/steering-committee should be the gatekeeper, and @howardjohn is on the committee.

**Comment by howardjohn** [comment]
fwiw that is my own opinion, if others on steering agree I am ok with this.

**Comment by zhlsunshine** [comment]
> I am not very comfortable with this blog post. It essentially is an advertisement for Kata containers and is almost completely unrelated to Istio. This is even explicitly called out in the conclusion:
> 
> "Basically, there is no any difference for using Kata Containers with other generic container runtimes in Istio."

Hi @howardjohn and @ericvn, I just want to give some introduction about Kata, I can make it be concise. And about the conclusion, I can add some constraints about it.

**Comment by howardjohn** [comment]
@zhlsunshine its not clear why this is relevant to Istio users or why this belongs on an Istio blog. Istio's blog is for content about Istio; this blog has little to do with Istio

**Review by howardjohn — COMMENTED** [review]
I reread the updated and I still don't see why this is beneficial to Istio users. It has nothing to do with Istio, we just deployed a custom Kubernetes setup and then... ran Istio in it. I think it's been fairly established that Istio runs in a variety of platforms.

Unless there is some deep integration I don't think there is value here *on Istio's blog*. Perhaps on another blog

**Comment by zhlsunshine** [comment]
Hi @howardjohn,  I removed unnecessary introduction and advertisement about Kata, then add some constraints for the first conclusion, is this okay?

**Comment by zhlsunshine** [comment]
Hi @howardjohn, okay, I may try to post this blog in Kata community ^_^

**Comment by kfaseela** [comment]
may be we could have added this in https://istio.io/latest/docs/setup/platform-setup/ ?

**Comment by rootsongjc** [comment]
> may be we could have added this in https://istio.io/latest/docs/setup/platform-setup/ ?

Kata Container is a kind of runtime, I don't think the platform setup page is the best place to post it. I suggest to post it to Kata Container's community.

**Pull request closed without merging**